tests/testthat/test_overall.R

```diff
@@ -236,6 +236,30 @@ test_that("Effects and Levels of Continuous Covariates", {
 
 })
 
+test_that("Collinear cols treated right", {
+
```

Comment:
Can you add a comment with the Stata command you did run (even if you have to pre-omit the covariate)?